pkg/oci/load_balancer.go

```diff
@@ -403,7 +406,7 @@ func (cp *CloudProvider) updateBackendSet(ctx context.Context, lbID string, acti
 		ports         = action.Ports
 	)
 
-	glog.V(2).Infof("Applying %q action on backend set %q for lb %q (ports=%+v)", action.Type(), action.Name(), lbID, ports)
+	cp.logger.Infof("Applying %q action on backend set %q for lb %q (ports=%+v)", action.Type(), action.Name(), lbID, ports)
```

Comment:
action stuff should be fields.

Reply:
lb id should be on the logger already. it gets to be super complex/annoying if we need to add the same fields every where down the call stack.